android/gradlew

```diff
@@ -1,78 +1,127 @@
-#!/usr/bin/env sh
+#!/bin/sh
```

Comment:
Can we migrate these gradle changes to a separate PR unless the changes are needed for this fix?

Reply:
Am open to this but I think even here it could suffice , but very open to migrate

Reply:
Done